microsoft/testsuites/performance/networkperf.py

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+from typing import Any, Dict, cast
+
+from lisa import (
+    Logger,
+    RemoteNode,
+    TestCaseMetadata,
+    TestSuite,
+    TestSuiteMetadata,
+    notifier,
+    simple_requirement,
+)
+from lisa.environment import Environment
+from lisa.features import Sriov, Synthetic
+from lisa.schema import NetworkDataPath
+from lisa.tools import Lagscope, Sysctl
+from lisa.util import dict_to_fields
+from microsoft.testsuites.network.common import stop_firewall
+from microsoft.testsuites.performance.common import cleanup_process
+
+
+@TestSuiteMetadata(
+    area="network",
+    category="performance",
+    description="""
+    This test suite is to validate linux network performance.
+    """,
+)
+class NetworkPerformace(TestSuite):
+    @TestCaseMetadata(
+        description="""
+        This test case uses lagscope to test synthetic network latency.
+        """,
+        priority=2,
+        requirement=simple_requirement(
+            min_count=2,
+            min_core_count=72,
+            network_interface=Synthetic(),
+        ),
+    )
+    def perf_tcp_latency_synthetic(self, environment: Environment) -> None:
+        self.perf_tcp_latency(environment, "perf_tcp_latency_synthetic")
+
+    @TestCaseMetadata(
+        description="""
+        This test case uses lagscope to test sriov network latency.
+        """,
+        priority=2,
+        requirement=simple_requirement(
+            min_count=2,
+            min_core_count=72,
```

Comment:
Does it need to limit the core count? This test case looks related to sriov only.

Reply:
I follow the same setting in lisav2, f72 will land into a dedicated host.

Reply:
What's the reason we have to run this case on f72? LISAv3 may use another vm size, which the core is more than 72. If there is no special reason, let's remove the core count.

Reply:
the normal size can't guarantee the performance, we choose the size which lands on dedicated host.

Reply:
ok, add into description to explain the test case design.

Reply:
The VM size should be selected when run the perf tests. If we're testing the kernel, it needs the high end vm size. But when we test the vm size itself, it doesn't need to specify it. So we should remove those settings, and add it to the perf pipelines.

Reply:
removed core count